src/game/shared/tf/tf_weapon_shotgun.cpp

```diff
@@ -329,27 +328,29 @@ void CTFScatterGun::FireBullet( CTFPlayer *pPlayer )
 			return;
 
 		// Knock the firer back!
-		if ( !(pOwner->GetFlags() & FL_ONGROUND) && !pPlayer->m_bScattergunJump )
+		if ( !(pOwner->GetFlags() & FL_ONGROUND) && !pPlayer->m_Shared.m_bScattergunJump )
 		{
-			pPlayer->m_bScattergunJump = true;
+			pPlayer->m_Shared.m_bScattergunJump = true;
 
+#ifndef CLIENT_DLL
 			pOwner->m_Shared.StunPlayer( 0.3f, 1.f, TF_STUN_MOVEMENT | TF_STUN_MOVEMENT_FORWARD_ONLY );
```

Comment:
Any reason we can't predict this?

Reply:
there isnt, i just imagined it could be a separate issue since a few other things (like thermal thruster) apply self-stun

i'll see if i can implement predicted self-stun for this pr as well